src/components/Accordion/__tests__/Accordion.spec.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @file Accordion tests.
+ * @copyright IBM Security 2020
+ */
+
+import { render } from '@testing-library/react';
+import React from 'react';
+
+import { carbonPrefix } from '../../../globals/namespace';
+
+import { Accordion, AccordionItem } from '../../..';
+
+describe('Accordion', () => {
+  test('should have icon aligned at start by default', () => {
+    const { getByLabelText, getByText } = render(
+      <Accordion>
+        <AccordionItem title="test title" iconDescription="test icon label">
+          test content
+        </AccordionItem>
+      </Accordion>
+    );
+    // Expect accordion to have correct alignment class:
+    expect(getByText(/test title/i).closest('ul')).toHaveClass(
+      `${carbonPrefix}accordion--start`
+    );
+    // Expect the icon to be first inside the accordion item heading button:
+    expect(getByText(/test title/i).closest('button').firstChild).toBe(
+      getByLabelText(/test icon label/i)
+    );
+  });
+
+  test('should pass custom class through spread attribute', () => {
```

Comment:
👏

Reply:
Just saw this comment https://github.com/carbon-design-system/ibm-security/pull/361#issuecomment-584843349 --

> My bad! One thing I realized, because we pass through ...other props, we should probably test for that too.

The test above `should pass custom class through spread attribute` checks `...other`. Since the custom class name isn't added to anymore / pull out, it's included in the spread attribute. That's why the test is worded kinda funny i.e., "pass custom class through spread attribute"

I guess technically it not required since the custom class is a Carbon thing, however writing the test is what made me realize that we weren't passing through custom classes. 😆

Reply:
Nice! Thanks, I just realised this too - I think this should cover it